### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by jrsewell400**
# Setup Rails app for Tips

This pull request adds all the functionality necessary in order to store tips in a database, and provide random tips when requested.  All components are tested and coverage is 100%.

## Type of change
Please uncheck options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Add Testing
- [ ] Refactor

## Checklist:
- [X] My code follows the Turing style guidelines
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: config/initializers/cors.rb [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
-#     origins 'example.com'
-#
-#     resource '*',
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins ''
```
Comment (alex-latham) [inline config/initializers/cors.rb:10]: We will specify the deployed heroku URL here once it's created

File: spec/requests/tips_request_spec.rb [hunk 2]
```diff
@@ -3,14 +3,15 @@
...
-    get '/api/vi/tips'
+    get '/api/v1/tips'
 
     expect(response).to be_successful
 
     parsed = JSON.parse(response.body, symbolize_names: true)
-    binding.pry
-    # expect(parsed.description)
+    
+    expect(tips).to include(parsed[:description])
```
Comment (PaulDebevec) [inline spec/requests/tips_request_spec.rb:15]: Well done with this expectation!

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -0,0 +1,101 @@
+
+require 'simplecov'
+SimpleCov.start do
+  add_filter 'spec/rails_helper.rb'
+end
```
Comment (alex-latham) [inline spec/spec_helper.rb:5]: This is very cool - didn't realize we could ignore files from coverage here.

File: config/initializers/cors.rb (hunk 1 above)
Comment (jrsewell400) [reply config/initializers/cors.rb:10]: Good catch!

File: spec/spec_helper.rb (hunk 3 above)
Comment (PaulDebevec) [reply spec/spec_helper.rb:5]: Yeah, sometimes it'll recognize your mailer (When you're not actually using the mailer), and this is a good way to skip that in the coverage

**Pull request merged**